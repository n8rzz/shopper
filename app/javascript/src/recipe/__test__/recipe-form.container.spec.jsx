import React from 'react';
import { shallow } from 'enzyme';
import {
    recipeIngredientMock,
    recipeInstructionMock,
    recipeMock,
} from '../../__mocks__/mocks';
import { RecipeFormContainer } from '../recipe-form.container';

describe('RecipeFormContainer', () => {
    test('renders correctly with valid props', () => {
        const component = shallow(<RecipeFormContainer
            recipe={recipeMock}
            ingredients={[recipeIngredientMock]}
            instructions={[recipeInstructionMock]}
            itemsById={{}}
        />);

        expect(component).toMatchSnapshot();
    });
});
