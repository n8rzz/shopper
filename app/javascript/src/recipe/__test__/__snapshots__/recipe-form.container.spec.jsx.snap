// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeFormContainer renders correctly with valid props 1`] = `
<div>
  RecipeFormContainer
</div>
`;
